pages/opam-using.html

```diff
@@ -94,15 +88,16 @@ <h3>Installing CoqIDE</h3>
 <p>You may also want to install CoqIDE. Note that this requires GTK+
 development files (<code>gtksourceview3</code>) to be available on the
 system. As for Coq, names of system packages to install can be
-determined using <code>opam-depext</code>:</p>
+determined using when running <code>opam install</code>:</p>
+
+<pre><code>opam install coqide
+</code></pre>
 
-<pre><code>opam-depext coqide</code></pre>
+<p><code>opam</code> will ask you if you want to continue with the installation, to which you can reply <code>n</code>.</p>
```

Comment:
OK, right, you can answer "n" and manually install the system package, but why would you do that? I guess if you can't use `sudo` to run your package manager (which should be a pretty edge case).

Reply:
Yeah, as I mentioned we should probably just make this simpler all together.